scripts/xcode_patcher.rb

```diff
@@ -175,6 +247,7 @@ def setup_target_extension(shortVersion, fullVersion, platform, configHash)
       config.build_settings['CLANG_ENABLE_MODULES'] ||= 'YES'
       config.build_settings['SWIFT_OBJC_BRIDGING_HEADER'] ||= 'macos/networkextension/WireGuardNetworkExtension-Bridging-Header.h'
       config.build_settings['SWIFT_PRECOMPILE_BRIDGING_HEADER'] = 'NO'
+      config.build_settings['APPLICATION_EXTENSION_API_ONLY'] = 'YES'
```

Comment:
Since I am touching this file I thought I might add this since it is required for XCode 13.